components/tooltip/tooltip.js

```diff
@@ -292,6 +291,7 @@ class Tooltip extends RtlMixin(LitElement) {
 
 	connectedCallback() {
 		super.connectedCallback();
+		this.showing = false;
```

Comment:
Worth adding a unit test that just creates the element using `createElement`?

Reply:
Going to add a test to all components in another PR